int32 motorIntensity
bool movingForwards
